### PR TITLE
v3 - replace current fonts with Britti sans

### PR DESCRIPTION
**Pull request by anemne**
## Short Description

This PR updates the current fonts to Britti Sans. The license permits up to 50,000 page views per month.

## Visual Overview (Image/Video)

Screenshot 2024-09-27 at 09 54 05

### CODE REVIEW

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -42,7 +35,7 @@ const preview: Preview = {
       }
 
       return (
-        <div className={`${fontFigtree.variable} ${fontRecoleta.variable}`}>
+        <div className={`${fontBrittiSans.variable}`}>
```
Comment (christinaroise) [inline .storybook/preview.tsx:38]: If we've only got one, I think we can do:
```suggestion
        <div className={fontBrittiSans.variable}>
```

File: src/app/layout.tsx [hunk 2]
```diff
@@ -45,7 +38,7 @@ export default async function RootLayout({
   }
   return (
     <html lang={siteLang}>
-      <body className={`${fontFigtree.variable} ${fontRecoleta.variable}`}>
+      <body className={`${fontBrittiSans.variable}`}>
```
Comment (christinaroise) [inline src/app/layout.tsx:41]:
```suggestion
      <body className={fontBrittiSans.variable}>
```

**Review by christinaroise — APPROVED** [review]

File: public/fonts/britti-sans-variable.woff2
Comment (mathiazom) [inline public/fonts/britti-sans-variable.woff2:1]: The file must be moved to `_assets`

File: .storybook/preview.tsx [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
-  src: "../../public/recoleta.otf",
-  variable: "--font-recoleta",
-});
-
-const fontFigtree = Figtree({
-  subsets: ["latin"],
-  weight: "400",
-  variable: "--font-figtree",
+const fontBrittiSans = localFont({
+  src: "../../public/fonts/britti-sans-variable.woff2",
```
Comment (mathiazom) [inline .storybook/preview.tsx:7]: Need to update folder here as well

**Pull request merged**